neuro-sdk/src/neuro_sdk/__init__.py

```diff
@@ -154,13 +154,20 @@
 
 
 def get(
-    *, path: Optional[Path] = None, timeout: aiohttp.ClientTimeout = DEFAULT_TIMEOUT
+    *,
+    path: Optional[Path] = None,
+    timeout: aiohttp.ClientTimeout = DEFAULT_TIMEOUT,
+    trace_configs: Optional[List[aiohttp.TraceConfig]] = None,
```

Comment:
Where the argument is used/passed?

Reply:
It will be used on server side. For example `platform-monitoring` used `neuro-sdk` to retrieve jobs

Reply:
Ok